store/storetest/user_store.go

```diff
@@ -203,28 +194,17 @@ func testUserStoreUpdate(t *testing.T, ss store.Store) {
 	require.Nil(t, err)
 
 	u3.Email = MakeEmail()
-	if userUpdate, err := ss.User().Update(u3, false); err != nil {
-		t.Fatal("Update should not have failed")
-	} else {
-		newUser := userUpdate.New
-		if newUser.Email != oldEmail {
-			t.Fatal("Email should not have been updated as the update is not trusted")
-		}
-	}
+	userUpdate, err := ss.User().Update(u3, false)
+	require.Nil(t, err, "Update should not have failed")
+	require.Equal(t, oldEmail, userUpdate.New.Email, "Email should not have been updated as the update is not trusted")
```

Comment:
This, for example, would be a better place to use `assert` instead of `require`

Reply:
Changed!